### PR TITLE
Add more languages to the Development category

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

**Review by Toasterson — APPROVED** [review]

File: src/util/opensolaris.org.sections [hunk 1]
```diff
@@ -31,7 +31,7 @@ category = Accessories,Configuration and Preferences,Games,Graphics and Imaging,
 category = Documentation,File Managers,Libraries,Localizations,Scripts,Sessions,Theming,Trusted Extensions,Window Managers
 
 [Development]
-category = C,C++,Databases,Distribution Tools,Editors,Fortran,GNOME and GTK+,GNU,High Performance Computing,Integrated Development Environments,Java,Objective C,Observability,Other Languages,PHP,Perl,Python,Ruby,Smalltalk,Source Code Management,Suites,System,X11
+category = Assembly language,C,C++,Databases,Distribution Tools,Editors,Erlang,Fortran,GNOME and GTK+,GNU,Go,Haskell,High Performance Computing,Integrated Development Environments,Java,JavaScript,Julia,Lisp,Lua,Objective C,Observability,OCaml,Other Languages,PHP,Perl,Python,Ruby,Rust,Smalltalk,Source Code Management,Suites,System,Tcl,X11
```
Comment (pyhalov) [inline src/util/opensolaris.org.sections]: Assembler?
Where do you want to use these classifications?

Comment (alarcher) [reply src/util/opensolaris.org.sections]: We have nasm, yasm, etc... We always default to "Other Languages".

Comment (pyhalov) [reply src/util/opensolaris.org.sections]: I'm fine with this change, but I honestly think it's no-op...

Comment (alarcher) [reply src/util/opensolaris.org.sections]: Sure I know it is just a bit of cosmetics :)

Comment (alarcher) [reply src/util/opensolaris.org.sections]: It is like having "Desktop (GNOME)" as category... it annoys me for some reason :S

**Pull request merged**